AutomaticInterface/AutomaticInterface/Builder.cs

```diff
@@ -105,7 +105,10 @@ private static void AddMethod(InterfaceBuilder codeGenerator, IMethodSymbol meth
         ActivateNullableIfNeeded(codeGenerator, method);
 
         var paramResult = new HashSet<string>();
-        method.Parameters.Select(GetMethodSignature).ToList().ForEach(x => paramResult.Add(x));
+        method
+            .Parameters.Select(x => x.ToDisplayString(FullyQualifiedDisplayFormat))
```

Comment:
The `FullyQualifiedDisplayFormat` can format the parameters for us, so we can now remove all the custom code for generating parameters (see the deletions of the `GetMethod...` methods below)